### PR TITLE
Grammar, spelling and styling edits

### PR DESCRIPTION
**Pull request by ab3lson**
## Proposed change
There were a few grammatical errors that were fixed. Capitalization of `Plex` was made uniform and other small styling tweaks were made.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/plex.markdown [hunk 1]
```diff
@@ -186,12 +188,12 @@ media_content_id: '{ "library_name": "Adult Movies", "title": "die hard", year=1
 
 ## Sonos Playback
 
-To play Plex music directly to Sonos speakers, the following requirements must be met:
+To play `Plex` music directly to Sonos speakers, the following requirements must be met:
```
Comment (frenck) [inline source/_integrations/plex.markdown]: We don't codefence brand names

File: source/_integrations/plex.markdown [hunk 2]
```diff
@@ -186,12 +188,12 @@ media_content_id: '{ "library_name": "Adult Movies", "title": "die hard", year=1
...
 
-1. Have an active [Plex Pass](https://www.plex.tv/plex-pass/) subscription.
-2. Remote access enabled for your Plex server.
-3. Sonos speakers linked to your Plex account [(Instructions)](https://support.plex.tv/articles/control-sonos-playback-with-a-plex-app/).
-4. [Sonos](/integrations/sonos/) integration configured.
+1. Have an active [Plex Pass](https://www.plex.tv/plex-pass/) subscription
+2. Remote access enabled for your Plex server
+3. Sonos speakers linked to your Plex account [(Instructions)](https://support.plex.tv/articles/control-sonos-playback-with-a-plex-app/)
+4. [Sonos](/integrations/sonos/) integration configured
```
Comment (frenck) [inline source/_integrations/plex.markdown]: Sentences have to have a full stop. Please restore the closing punctuation.

File: source/_integrations/plex.markdown [hunk 3]
```diff
@@ -254,5 +256,5 @@ play_plex_on_tv:
 
 ## Notes
 
-* The `plex` integration supports multiple Plex servers. Additional connections can be configured under **Configuration** > **Integrations**.
-* Movies must be located under 'Movies' section in the Plex library to properly get 'playing' state.
+* The `Plex` integration supports multiple Plex servers. Additional connections can be configured under **Configuration** > **Integrations**.
```
Comment (frenck) [inline source/_integrations/plex.markdown]:
```suggestion
* The Plex integration supports multiple Plex servers. Additional connections can be configured under **Configuration** > **Integrations**.
```

File: source/_integrations/plex.markdown [hunk 4]
```diff
@@ -254,5 +256,5 @@ play_plex_on_tv:
...
+* Movies must be located under the 'Movies' section in a `Plex` library to properly view the 'playing' state.
```
Comment (frenck) [inline source/_integrations/plex.markdown]:
```suggestion
* Movies must be located under the 'Movies' section in a Plex library to properly view the 'playing' state.
```

File: source/_integrations/plex.markdown [hunk 5]
```diff
@@ -57,14 +59,14 @@ Alternatively, you can manually configure a Plex server connection by selecting
...
 
 
 ## Media Player
 
-The `plex` media_player platform will create Media Player entities for each connected client device. These entities will display media information, playback progress, and playback controls if supported by the device.
+The `Plex` media_player platform will create Media Player entities for each connected client device. These entities will display media information, playback progress, and playback controls (if supported by the streaming device).
 
-By default the Plex integration will create Media Player entities for all local, managed, and shared users on the Plex server. To customize which users or client types to monitor, adjust the "*Monitored users*", "*Ignore new managed/shared users*", and "*Ignore Plex Web clients*" options described under [Integration Options](#integration-options).
+By default the `Plex` integration will create Media Player entities for all local, managed, and shared users on the Plex server. To customize which users or client types to monitor, adjust the "*Monitored users*", "*Ignore new managed/shared users*", and "*Ignore Plex Web clients*" options described under [Integration Options](#integration-options).
```
Comment (frenck) [inline source/_integrations/plex.markdown]:
```suggestion
By default, the Plex integration will create Media Player entities for all local, managed, and shared users on the Plex server. To customize which users or client types to monitor, adjust the "*Monitored users*", "*Ignore new managed/shared users*", and "*Ignore Plex Web clients*" options described under [Integration Options](#integration-options).
```

File: source/_integrations/plex.markdown [hunk 6]
```diff
@@ -13,18 +13,20 @@ ha_codeowners:
...
 There is currently support for the following device types within Home Assistant:
 
 - [Sensor](#sensor)
 - [Media Player](#media-player)
 
-If your Plex server has been claimed by a Plex account via the [claim interface](https://plex.tv/claim), Home Assistant will require authentication to connect.
+If a Plex server has been claimed by a Plex account via the [claim interface](https://plex.tv/claim), Home Assistant will require authentication to connect.
 
-The Plex integration is set up via **Configuration** -> **Integrations**. You will be redirected to the [Plex](https://plex.tv) website to sign in with your Plex account. Once access is granted, Home Assistant will connect to the server linked to the associated account. If multiple Plex servers are available on the account, you will be prompted to complete the configuration by selecting the desired server on the Integrations page. Home Assistant will show as an authorized device on the [Plex Web](https://app.plex.tv/web/app) interface under **Settings** -> **Authorized Devices**.
+### Automatic Configuration
```
Comment (frenck) [inline source/_integrations/plex.markdown]:
```suggestion
### Configuration
```

File: source/_integrations/plex.markdown [hunk 7]
```diff
@@ -13,18 +13,20 @@ ha_codeowners:
 ha_domain: plex
 ---
 
-The `plex` integration allows you to connect to a [Plex Media Server](https://plex.tv). Once connected, [Plex Clients](https://www.plex.tv/apps-devices/) playing media from the connected Plex Media Server will show up as [Media Players](/integrations/media_player/) and report playback status via a [Sensor](/integrations/sensor/) in Home Assistant. The Media Players will allow you to control media playback and see the current playing item.
+The `Plex` integration allows you to connect Home Assistant to a [Plex Media Server](https://plex.tv). Once configured, actively streaming [Plex Clients](https://www.plex.tv/apps-devices/) show up as [Media Players](/integrations/media_player/) and report playback status via a [Sensor](/integrations/sensor/) in Home Assistant. Plex Media Players will allow you to control media playback and see the current playing item.
```
Comment (frenck) [inline source/_integrations/plex.markdown]:
```suggestion
The Plex integration allows you to connect Home Assistant to a [Plex Media Server](https://plex.tv). Once configured, actively streaming [Plex Clients](https://www.plex.tv/apps-devices/) show up as [Media Players](/integrations/media_player/) and report playback status via a [Sensor](/integrations/sensor/) in Home Assistant. Media Players will allow you to control media playback and see the current playing item.
```

The latter media player, refers to the platform, not Plex.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR, @ab3lson!

I've left some comments, could you please take a look?

Thanks! 👍

File: source/_integrations/plex.markdown (hunk 2 above)
Comment (ab3lson) [reply source/_integrations/plex.markdown]: Revised

**Review by frenck — APPROVED** [review]
Thanks, @ab3lson 👍 

Happy Hacktoberfest 🎉

**Pull request merged**